roles/neutron-common/templates/usr/local/bin/neutron-reset

```diff
@@ -285,5 +293,10 @@ class ResetNeutron(object):
             return False
 
 
+parser = OptionParser()
+parser.add_option("-d", "--device", type="string", dest="device", help="devices to be reset. eg. vxlan,bridge,tap")
+
+(options, args) = parser.parse_args()
```

Comment:
You're technically using these as globals, which I personally dislike because you're already using classes, which achieve the same function - which is to have namespaced state.

Globals should be capitalized per PEP8 but I'd go one step further and remove them if feasible.

Consider explicitly passing your args directly into the class if possible.

Additionally, I'd move this entire block of code into the `if __name__ == '__main__'` statement below, so that this code is not run as a side effect if this script is ever to be used as an imported module in the future.